src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  &__title {
+    padding-inline: 16px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    position: absolute;
+    top: 206px;
+    transition-duration: $trans-duration;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__product-code {
+    padding-inline: 16px;
+    position: absolute;
+    top: 246px;
```

Comment:
Similar to the card title, absolute positioning for the product code can lead to layout problems. Evaluate if a more flexible layout approach could be beneficial.